### PR TITLE
Fix GitHub Actions

### PR DESCRIPTION
**Pull request by theLastOfCats**
updated NodeJS in GH Actions to current LTS and latest

use pip3 in GH Actions

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,28 +1,28 @@
 name: Build
 
-on: [push]
+on: [push, pull_request]
```
Comment (sobolevn) [inline .github/workflows/build.yml:3]: Please, add `workflow_dispatch`

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks!

File: .github/workflows/build.yml (hunk 1 above)
Comment (theLastOfCats) [reply .github/workflows/build.yml]: added

**Review by sobolevn — APPROVED** [review]
Thanks a lot!

**Comment by theLastOfCats** [comment]
Think now you can start bump deps from dependabot :)

**Pull request merged**